packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -291,8 +291,11 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     }
 
     /**
-     * @notice Instantly relay a deposit amount minus fees. Instant relayer earns a reward following the pending relay
-     * challenge period.
+     * @notice Instantly relay a deposit amount minus fees to the recipient. Instant relayer earns a reward following
+     * the pending relay challenge period.
+     * @dev We assume that the caller has performed an off-chain check that the relayed deposit data is valid that they
```

Comment:
@chrismaree wdyt about this way to describe the instant relayers responsibilities?